apm-webapp/pom.xml

```diff
@@ -70,11 +70,6 @@
             <artifactId>spring-boot-configuration-processor</artifactId>
             <version>${spring.boot.version}</version>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-devtools</artifactId>
-            <version>${spring.boot.version}</version>
-        </dependency>
```

Comment:
@wu-sheng  It turns out that this is the reason why I said `webapp` doesn't support JDK 9, it's this module (`devtool`) that doesn't support JDK9, and it's just for development, so removing it does no harm

Reply:
Yes, this should be safe to be removed. This is byte code manipulated tech :) So it needs to do many things to be compatible with various JDKs :P